### PR TITLE
Add hackathon result

### PR DESCRIPTION
**Pull request by mkubdev**
#### Description of the change

Add results to readme.

#### Notes

Added screenshot.

[contributing]: https://github.com/web3community/DEV-NFT/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Review by KukretiShubham — APPROVED** [review]

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 
 This project aims for [Buidlit](https://buidlit.polygon.technology/) & [Dev Dapp Starter Grant](https://www.notion.so/Welcome-to-DEV-DAPP-STARTER-GRANTS-5cb95252f18540258111581ea54d8808)
 
+
+# Hackaton results
+
+This project win 2 prizes for [Buidlit](https://buidlit.polygon.technology/) hackaton 🏆 :
```
Comment (krishguptadev) [inline README.md:8]:
```suggestion
This project won 2 prizes for [Buidlit](https://buidlit.polygon.technology/) hackathon 🏆 :
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
 
 This project aims for [Buidlit](https://buidlit.polygon.technology/) & [Dev Dapp Starter Grant](https://www.notion.so/Welcome-to-DEV-DAPP-STARTER-GRANTS-5cb95252f18540258111581ea54d8808)
 
+
+# Hackaton results
```
Comment (krishguptadev) [inline README.md:6]:
```suggestion
# Hackathon results
```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+
+This project win 2 prizes for [Buidlit](https://buidlit.polygon.technology/) hackaton 🏆 :
+
+<img width="1259" alt="Capture d’écran 2021-12-27 à 16 15 30" src="https://user-images.githubusercontent.com/50140834/147484853-81c5bae0-8a8a-40d8-bdcb-f50b795f59d1.png">
+
+Check the project submission [here](https://devfolio.co/submissions/devnft-210b) !
```
Comment (krishguptadev) [inline README.md:12]:
```suggestion
Check the project submission [here](https://devfolio.co/submissions/devnft-210b)!
```

File: README.md [hunk 4]
```diff
@@ -2,6 +2,17 @@
 
 This project aims for [Buidlit](https://buidlit.polygon.technology/) & [Dev Dapp Starter Grant](https://www.notion.so/Welcome-to-DEV-DAPP-STARTER-GRANTS-5cb95252f18540258111581ea54d8808)
 
+
+# Hackaton results
+
+This project win 2 prizes for [Buidlit](https://buidlit.polygon.technology/) hackaton 🏆 :
+
+<img width="1259" alt="Capture d’écran 2021-12-27 à 16 15 30" src="https://user-images.githubusercontent.com/50140834/147484853-81c5bae0-8a8a-40d8-bdcb-f50b795f59d1.png">
```
Comment (krishguptadev) [inline README.md:10]:
```suggestion
<img width="1259" alt="" src="https://user-images.githubusercontent.com/50140834/147484853-81c5bae0-8a8a-40d8-bdcb-f50b795f59d1.png">
```

**Review by KukretiShubham — APPROVED** [review]
LGTM

**Review by shuklaritvik06 — APPROVED** [review]
LGTM 🎉

**Pull request merged**